lib/core/src/display/navigation/mouse_manager.rs

```diff
@@ -0,0 +1,315 @@
+use crate::display::render::css3d::DOMContainer;
```

Comment:
Btw, let's agree that on top of each file we will write a short documentation telling what this file contains and how it is used. After few PRs I'm really, really missing such docs, as borwsing the inside of the file and all implementations to get the grasp of "what it is about" is very time consuming. If you agree, I would add it as a request to our style guide. CC: @farmaazon @mwu-tow

Reply:
I think it's a good idea. Can we establish a format?

Reply:
Yep, In my next PR I will add everywhere `warn(missing_docs)` (see the Discord ide channel for more info). It just guards us to put docs where necessary automatically! :)